src/main/java/frc/robot/subsystems/CoralArm.java

```diff
@@ -38,23 +43,29 @@ public CoralArm() {
         config.closedLoop
                 .pid(RobotMap.ARM_CORAL_KP, RobotMap.ARM_CORAL_KI, RobotMap.ARM_CORAL_KD)
                 .iZone(RobotMap.ARM_CORAL_IZONE)
-                .outputRange(-1, 1);
+                .outputRange(-1, 1)
+                .feedbackSensor(ClosedLoopConfig.FeedbackSensor.kPrimaryEncoder);
+        config.idleMode(SparkBaseConfig.IdleMode.kBrake);
```

Comment:
you also need to configure the encoder with the gear ratio information so that it knows to convert values accordingly. Use

```
config.encoder
    .positionConversionFactor(1 / GEAR_RATIO)
    .velocityConversionFactor(1 / GEAR_RATIO)
```